File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TreeViews_ViewMyParentIndexRowHandlesRow.cpp

#include "TreeViews_ViewMyParentIndexRowHandlesRow.h"
#include "TreeViews_ViewMyParentIndexRowHandlesRow.h"
#include <string>

namespace widgetassertions
{
  std::string TreeViews_ViewMyParentIndexRowHandlesRow::getValuesLabelText( ) 
  {
    return this->ValuesLabelText;
  }
  int TreeViews_ViewMyParentIndexRowHandlesRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
  int TreeViews_ViewMyParentIndexRowHandlesRow::getParentRowIndex( ) 
  {
    return this->ParentRowIndex;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TreeViews_ViewMyParentIndexRowHandlesRow.cs
/// <filename>
///     TreeViews_ViewMyParentIndexRowHandlesRow.cs
/// </filename>


namespace widgetassertions
{
    public class TreeViews_ViewMyParentIndexRowHandlesRow
  {
        private string ValuesLabelText;
     
        public     virtual string getValuesLabelText()
    {
      return this.ValuesLabelText;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
     
        private int ParentRowIndex;
     
        public     virtual int getParentRowIndex()
    {
      return this.ParentRowIndex;
    }
  }
}

